apps/condo/domains/miniapp/components/AppSelector/AppSelectPageContent.tsx

```diff
@@ -0,0 +1,141 @@
+                                    url={'/settings?tab=billing'}
+                                />
+                                <AppCarouselCard
+                                    logoSrc={'https://logo.clearbit.com/amazon.com'}
+                                    title={'Слушай, что-то я похоже очень большой текст'}
+                                    url={'https://www.shopify.com'}
+                                />
+                                <AppCarouselCard
+                                    logoSrc={'https://logo.clearbit.com/asos.com'}
+                                    title={'Слушай, что-то я похоже очень большой текст'}
+                                    url={'https://www.google.com'}
+                                />
+                                <AppCarouselCard
+                                    logoSrc={'https://logo.clearbit.com/ebay.com'}
+                                    title={'Слушай, что-то я похоже очень большой текст'}
+                                    url={'https://www.spotify.com'}
+                                />
+                                <AppCarouselCard
+                                    logoSrc={'https://logo.clearbit.com/apple.com'}
+                                    title={'Слушай, что-то я похоже очень большой текст'}
+                                    url={'/settings?tab=billing'}
+                                />
+                            </Carousel>
+                        </Section>
+                    </Col>
+                }
+                <Col span={24}>
+                    <Section title={AvailableSectionTitle} hideTitle={!isAnyServiceConnected}>
+                        <CardsContainer cardsPerRow={servicesPerRow}>
+                            <AppSelectCard
+                                logoSrc={'https://logo.clearbit.com/shopify.com'}
+                                title={'Слушай, что-то я похоже очень большой текст'}
+                                url={'https://www.shopify.com'}
+                                description={'loremloremloremloremloremlorem loremloremlorem loremloremloremlorem loremloremloremloremlorem loremloremloremloremloremlorem'}
+                            />
+                            <AppSelectCard
+                                title={'Я без логотипа'}
+                                url={'https://www.shopify.com'}
+                                description={'Короткое описание'}
+                            />
+                            <AppSelectCard
+                                title={'Я disabled'}
+                                url={'https://www.shopify.com'}
+                                description={'Короткое описание'}
+                                disabled
+                            />
+                            <AppSelectCard
+                                title={'У меня есть тэг'}
+                                tag={'Крутой тэг'}
```

Comment:
No translations here

Reply:
It's just a demonstration....will be removed

Reply:
Already removed....https://github.com/open-condo-software/condo/pull/1277